ambari-server/src/main/java/org/apache/ambari/server/events/publishers/AgentCommandsPublisher.java

```diff
@@ -95,7 +96,7 @@ public class AgentCommandsPublisher {
 
   public void sendAgentCommand(Multimap<Long, AgentCommand> agentCommands) throws AmbariRuntimeException {
     if (agentCommands != null && !agentCommands.isEmpty()) {
-      Map<Long, TreeMap<String, ExecutionCommandsCluster>> executionCommandsClusters = new TreeMap<>();
+      Map<Long, TreeMap<String, ExecutionCommandsCluster>> executionCommandsClusters = new ConcurrentHashMap<>();
       Map<Long, Map<String, DesiredConfig>> clusterDesiredConfigs = new HashMap<>();
```

Comment:
@dmmkr 
How about using `ConcurrentHashMap` for `clusterDesiredConfigs` also?

Reply:
Thanks @eubnara for the review.
I have updated clusterDesiredConfigs to use ConcurrentHashMap